docs: multiple ports docs for manifest (#4684)
aws/copilot-cli: site/content/docs/manifest/backend-service.en.md

              efs:
                id: fs-1234567
        ```

>         === "Multiple Ports Exposed"

Reviewer: Not sure which name is more interesting and descriptive 🤔  Throwing some ideas here

```suggestion
        === "Expose multiple ports"
```

```suggestion
        === "Multiple paths"
```

        ```yaml
        name: 'backend'
        type: 'Backend Service'

        image:

>           build: './frontend/Dockerfile'

Reviewer: ```suggestion
          build: './backend/Dockerfile'
```

          port: 8080

        http:
          path: '/'

>           target_port: 8083 // exposed port 8083 of the main container i.e. backend on the path "/" 

Reviewer: ```suggestion
          target_port: 8083 // Traffic on "/" is forwarded to the main container, on port 8083.
```

          additional_rules:

>             - target_port: 8081 // exposed port 8081 of the main container i.e. backend on the path "customerdb"

Reviewer: ```suggestion
            - target_port: 8081 // Traffic on "/customerdb" is forwarded to the main container, on port 8081.
```

              path: 'customerdb'

>             - target_port: 8082 // exposed port 8082 of the sidecar container i.e. envoy on the path "admin"

Reviewer: ```suggestion
            - target_port: 8082 // Traffic on "/admin" is forwarded to the sidecar "envoy", on port 8082.
```

The HTTP(S) protocol version. Must be one of `'grpc'`, `'http1'`, or `'http2'`. If omitted, then `'http1'` is assumed.
If using gRPC, please note that a domain must be associated with your application.

<span class="parent-field">http.</span><a id="http-additional-rules" href="#http-additional-rules" class="field">`additional_rules`</a> <span class="type">Array of Maps</span>  

> The additional listener rules allows you to configure multiple ALB listener rules.

Reviewer 2: ```suggestion
Configure multiple ALB listener rules.
```
